src/interactions/scrollInteraction.ts

```diff
@@ -0,0 +1,58 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Interaction {
+
+  export type ScrollCallback = (p: Point, deltaAmount: number) => any;
+
+  export class Scroll extends Interaction.AbstractInteraction {
+
+    /**
+     * The number of pixels occupied in a line.
+     */
+    public static PIXELS_PER_LINE = 120;
```

Comment:
Based on this post http://stackoverflow.com/questions/20110224/what-is-the-height-of-a-line-in-a-wheel-event-deltamode-dom-delta-line

it's not consistent.  Thus, D3 is basically using a guess estimate.  We can stick with the estimate to get behavior similar to D3 or we can come up with our own logic.